lighthouse-cli/bin.js

```diff
@@ -94,6 +94,12 @@ async function begin() {
     cliFlags.outputPath = 'stdout';
   }
 
+  // show deprecation message for disableDeviceEmulation
+  if (cliFlags.disableDeviceEmulation) {
```

Comment:
I think you'll have to `// @ts-ignore disableDeviceEmulation has been removed from cliFlags` this line now that #8289 has removed it